script/Ciel/Day 5/05_01_CIEL05_1E.txt

```diff
@@ -21,13 +21,14 @@ The ceiling is supported by numerous pillars.
 [sha:b027beb5f34b748e687fc98331aac8d9992c0e32]{
 -- Page 199, Offset 11399.
 -- 天井中心部には窓らしき穴があり、灰色の陽差しを下ろしている。それこそ、雲の隙間からのぞく<天使の梯子|ジェイコブズラダー>のように。
-In the center of the ceiling, there's a window-like hole from which the greyish sunlight fell. It's exactly like a <sunbeam|J a c o b ' s  l a d d e r> peeking through gaps in the clouds.
+// what the shit is jacobs ladder
```

Comment:
this is a double whammy of Nasu's beloved superficial comparative theology crap and some wacky foreign term to trip up the native readers. [Jacob's Ladder](https://www.koco.com/article/rays-of-sunlight-known-as-jacob-s-ladder/4300753) is both a phenomenon where light shines through clouds and a biblical allusion